src/js/jsx/search/SearchBar.jsx

```diff
@@ -230,9 +230,19 @@ define(function (require, exports, module) {
                         return toRemove ? toRemove.replace(" ", "") : idWord;
                     }, this).join("").toLowerCase(),
                     nextFilterMap = _.map(currFilter, function (word) {
-                        return idString.includes(word.toLowerCase()) ? "" : word;
+                        if (idString.includes(word.toLowerCase())) {
+                            partialMatch = true;
```

Comment:
Map functions should always return a value. It's weird that this function only returns a value sometimes. If this is intentional, you should explicitly return `null` instead of having no return.